.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -83,14 +83,16 @@ This PR affects the following parts of the plugin, which may require extra testi
 ## Quality assurance
 
 * [ ] I have tested this code to the best of my abilities.
-* [ ] During testing, I had activated all plugins Yoast SEO provides integrations for.
+* [ ] During testing, I had activated all plugins Yoast SEO provides integrations for. [List](https://github.com/Yoast/wordpress-seo/blob/trunk/readme.txt#L106)
 * [ ] I have added unit tests to verify the code works as intended.
 * [ ] If any part of the code is behind a feature flag, my test instructions also cover cases where the feature flag is switched off.
 * [ ] I have written this PR in accordance with my team's definition of done.
+* [ ] I have checked that base branch is correctly set. May be other than *trunk* branch is better alternative.
 
 ## Innovation
 
 * [ ] No innovation project is applicable for this PR.
 * [ ] This PR falls under an innovation project. I have attached the `innovation` label and noted the work hours.
```

Comment:
```suggestion
* [ ] This PR falls under an innovation project. I have attached the `innovation` label.
```
Since we now have a separate check box for noting down the work hours in the WBSO document, this phrase is redundant. I would remove it.